scripts/library/input.js

```diff
@@ -0,0 +1,35 @@
+//@ts-check
+/** @typedef {HTMLInputElement | null} HTMLInput*/
+export default class InputHandler {
+
+  /** <input> 요소의 값이 비었는지 확인
+   * @param {string} query - cssSelector
+   * */
+  static isEmptyValue = query => {
+    /** @type {HTMLInput} */
+    const element = document.querySelector(query);
+    if (element?.tagName === 'INPUT') return element.value.trim() === '' ? true : false;
```

Comment:
아래와 같이 코드를 줄일 수도 있을 것 같아요 
```suggestion
    if (element?.tagName === 'INPUT') return element.value.trim() === '';
```

Reply:
정정했습니다! https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/pull/175/commits/49e637b86fff13c8c2f773f9eca402f139d442d2